projects/storyteller/webapp/src/app/app.module.ts

```diff
@@ -27,17 +27,12 @@ import { SingleScreenshotComponent } from './components/single-screenshot/single
     FirebaseModule,
     AppRoutingModule,
     BrowserAnimationsModule,
-    MatToolbarModule,
-    MatIconModule, 
-    MatButtonModule,
-
-  ],
-  exports: [
     MatToolbarModule,
     MatIconModule,
     MatButtonModule,
 
   ],
+  exports: [],
```

Comment:
Great! But not need to stay module fields, if they are empty.
So we can delete `exports` and `providers` at all.